cernopendata/mappings/v7/records/record-v1.0.0.json

```diff
@@ -1,6 +1,9 @@
 {
     "mappings": {
       "properties": {
+        "doi": {
+          "type":"text"
```

Comment:
Which problem are you addressing in this PR?  Searching for `10.7483/OPENDATA.OPERA.NRED.999W` wouldn't give good results still...

Reply:
@tiborsimko This will work after the merge of #3054.

Reviewing/Merging can go like this:

-> #3049 
-> #3054
-> #3058

Reply:
Please rebase 3054 against latest `master` and I'll look there first

Reply:
query | opendata.cern.ch (production) | PR 3058
----- |  ----------------------------- | -------
10.7483/OPENDATA.9S5F.BY3B | error | 1 result
"10.7483/OPENDATA.9S5F.BY3B" | 1 result | 1 result
10.7483 | 2123 results | 2126 results

@tiborsimko This is the updated results after the PR #3054